guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -0,0 +1,365 @@
+Enterprise Linux:: An umbrella term for the following {RHEL}-like operating systems:
+
+ AlmaLinux
+ CentOS Linux
+ CentOS Stream
+ Oracle Linux
+ {RHEL}
+ Rocky Linux
+
+ifndef::orcharhino[]
+{Project} is tested on AlmaLinux and CentOS Stream.
+endif::[]
+endif::[]
+
+[[ERB]]
+ERB:: Embedded Ruby (ERB) is a template syntax used in provisioning and job templates.
+
+[[Errata]]
+Errata:: Updated RPM packages containing security fixes, bug fixes, and enhancements.
+In relationship to a host, erratum is *applicable* if it updates a package installed on the host and *installable* if it is present in the host's content view (which means it is accessible for installation on the host).
+
+[[External_Node_Classifier]]
+External Node Classifier:: A construct that provides additional data for a server to use when configuring hosts.
+{ProjectName} acts as an External Node Classifier to Puppet servers in a {Project} deployment.
++
+Note that the External Node Classifier will be removed in the next {Project} version.
+
+[[Facter]]
+Facter:: A program that provides information (facts) about the system on which it is run; for example, Facter can report total memory, operating system version, architecture, and more.
+Puppet modules enable specific configurations based on host data gathered by Facter.
+
+[[Facts]]
+Facts:: Host parameters such as total memory, operating system version, or architecture.
+Facts are reported by Facter and used by Puppet.
+
+[[Foreman]]
+Foreman:: The component mainly responsible for provisioning and content lifecycle management.
+ifdef::satellite[]
+Foreman is the main upstream counterpart of {ProjectName}.
+endif::[]
+
+[[{project-context}_Services]]
+{Project} services:: A set of services that {ProjectServer} and {SmartProxyServers} use for operation.
+You can use the `{foreman-maintain}` tool to manage these services.
+To see the full list of services, enter the `{foreman-maintain} service list` command on the machine where {Project} or {SmartProxyServer} is installed.
+
+[[Foreman_Hook]]
+Foreman Hook:: An executable that is automatically triggered when an orchestration event occurs, such as when a host is created or when provisioning of a host has completed.
+ifdef::satellite[]
++
+Note that Foreman Hook functionality is deprecated and will be removed in the next {Project} version.
+endif::[]
+
+[[Full_Host_Image]]
+Full Host Image:: A boot disk used for PXE-less provisioning of a specific host.
+The full host image contains an embedded Linux kernel and init RAM disk of the associated operating system installer.
+
+[[Generic_Image]]
+Generic Image:: A boot disk for PXE-less provisioning that is not tied to a specific host.
+The generic image sends the host’s MAC address to {ProjectServer}, which matches it against the host entry.
+
+[[Hammer]]
+Hammer:: A command line tool for managing {ProjectName}.
+You can execute Hammer commands from the command line or utilize them in scripts.
+Hammer also provides an interactive shell.
```

Comment:
Just a general observation/challenge on the usefulness of glossaries: Here, it would be a good idea to add a link to the (RH d/s) Hammer guide or a hint to check out the man page. But then again, why not add such a reference directly to modules where users might be wondering what Hammer is?

Reply:
I have created a separate reference for CLI things. I would also like to bring https://docs.orcharhino.com/or/docs/sources/installation_and_maintenance/using_hammer_cli.html upstream someday.

In general: Yes, we need to (and we do!) provide the necessary information and subcommands in procedures, not the glossary.